builder/vmware/iso/config.go

```diff
@@ -160,16 +150,8 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 		}
 	}
 
-	if c.Network == "" {
-		c.Network = "nat"
-	}
-
-	if !c.Sound {
-		c.Sound = false
-	}
-
-	if !c.USB {
-		c.USB = false
+	if c.HWConfig.Network == "" {
+		c.HWConfig.Network = "nat"
```

Comment:
Any reason why you've chosen not to put this into the HWConfig's Prepare() with the other defaulting?

Reply:
So the reason why I didn't consolidate the setting of the network to "nat" is because defaulting it as "nat" only made sense to me for the vmware-iso builder. Wrt the vmware-vmx builder, it'd be better to pull the default from the vmx itself and then change it if the user specifies the network in the template configuration.

But, this still doesn't support vmware-vmx at the moment. This huge HWConfig change that I neglected to mention (heh!) keeps all of the generic vmware junk inside HWConfig so this way if there's anything that's common between both the vmware-iso and vmware-vmx builders (such as cpus, memory, and this other stuff) it can be modified to expose options to all the builders.